tests/ui_tests/app/page/cluster/page.py

```diff
@@ -445,6 +448,50 @@ def delete_host_by_row(self, row: WebElement):
     def check_cluster_hosts_toolbar(self, cluster_name: str, host_name: str):
         self.toolbar.check_toolbar_elements(["CLUSTERS", cluster_name, "HOSTS", host_name])
 
+    @allure.step('Click on maintenance mode button in row {row_num}')
+    def click_on_maintenance_mode_btn(self, row_num: int):
+        """Click maintenance mode in row"""
+
+        row = self.table.get_row(row_num)
+        self.find_child(row, ClusterHostLocators.HostTable.HostRow.maintenance_mode_btn).click()
+
+    @allure.step('Assert maintenance mode state in row {row_num}')
+    def assert_maintenance_mode_state(self, row_num: int, is_mm_state_on: Optional[bool] = True):
+        """
+        Assert maintenance mode state in row
+        :param row_num: number of the row with maintenance mode button
+        :param is_mm_state_on: state of maintenance mode button:
+            True for ON state
+            False for OFF state
+            None for not available state
+        """
+
+        def _check_mm_state(page: ClusterHostPage, row: WebElement):
```

Comment:
since you define this function inside another, you can omit passing page and row, but get them from the method scope directly